tsdb/store.go

```diff
@@ -326,6 +368,8 @@ func (s *Store) Open() error {
 		return err
 	}
 
+	go s.periodicMaintenance()
```

Comment:
it doesn't need it, the underlying engine won't close while maintenance is going on, which is what the wg is for